sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -283,6 +292,40 @@ public void captureUserFeedback(final @NotNull UserFeedback userFeedback) {
     }
   }
 
+  @Override
+  public void captureMetrics(final @NotNull EncodedMetrics metrics) {
+    if (!isEnabled()) {
+      options
+          .getLogger()
+          .log(
+              SentryLevel.WARNING,
+              "Instance is disabled and this 'captureMetrics' call is a no-op.");
+    } else {
+      final StackItem item = stack.peek();
+      final SentryEnvelopeItem envelopeItem = SentryEnvelopeItem.fromMetrics(metrics);
+
+      // TODO fine this way?
```

Comment:
Up for discussion, IMHO it's fine this way, other SDKs like python do it in a similar way.

Reply:
I'd keep it consistent with the other methods, so i'd move it into the client 😕